### ISSUE TITLE
[Bug] In dark mode, status bar pane dividers become white.

### ISSUE THREAD

**Issue by DirtyJohny** [Maplespe/ExplorerBlurMica#130]
![Explorer](https://github.com/Maplespe/ExplorerBlurMica/assets/73193639/0e0e4898-4a8a-4563-83a8-086656c9e5c2)
Sometimes a bug appears with a white stripe in the status bar (If you open a folder). If you minimize and expand the window, the bug disappears.

**Comment by DirtyJohny** [comment]
And thank you very much for the project, it’s a lot work, I really appreciate your contribution.

**Comment by aimusicradio** [comment]
Yes, this happens when I click open folder for a downloaded file from chrome
